app/renderer/src/containers/result-details-container/result-details-container.js

```diff
@@ -0,0 +1,41 @@
+import { ipcRenderer } from 'electron';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import ResultDetails from '../../components/result-details';
+import { IPC_ITEM_DETAILS_RESPONSE } from '../../../../ipc';
+import { ThemeSchema } from '../../schema';
+
+export default class ResultDetailsContainer extends Component {
+  static displayName = 'ResultDetailsContainer';
+
+  componentDidMount() {
+    ipcRenderer.on(IPC_ITEM_DETAILS_RESPONSE, (evt, details) => {
+      if (details) {
+        this.props.onLoad(details);
```

Comment:
interesting that prettier didn't kick here, and this would have been 

```js
if (foo)
  this.props.bar()

// OR

if (foo) this.props.bar()
```